README.md

```diff
@@ -86,6 +86,41 @@ repositories for example usage of `element-data-loader`.
      loaded_dataset = suite2p_loader.Suite2p(output_dir)
      ```
 
+
++ Suite2p functions for processing
+     
+     Each step of the Suite2p pipeline (registration, cell detection and 
+     deconvolution) can be run independently. The functions will facilitate this
+     process. The user will need to define the ops and the db dictionary before
+     one or all of the processing steps of Suite2p are run.
+
```

Comment:
+ These wrapper functions are required primarily because `run_s2p` cannot individually run deconvolution using the `spikedetect` flag ([Suite2p Issue #718](https://github.com/MouseLand/suite2p/issues/718)).